spark-ml/Dockerfile

```
@@ -0,0 +1,94 @@
+FROM srcd/spark:2.3.0
```

spark: 2.3.0 hasn't been build yet

---

```suggestion
FROM srcd/spark:2.2.3
```

Little typo (which i also mad on my local build), oops